amf-cli/shared/src/test/scala/amf/emit/CompatibilityCycleTest.scala

```diff
@@ -22,11 +22,25 @@ class CompatibilityCycleTest extends AsyncFunSuite with FileAssertionTest {
   private val basePath = "amf-cli/shared/src/test/resources/compatibility/"
 
   test("RAML with union of nil + type to OAS 3.0") {
-    compatibility("raml10/union-nil-plus-element.raml", "oas30/union-nil-plus-element.yaml", Raml10YamlHint, Oas30YamlHint)
+    compatibility(
+      "raml10/union-nil-plus-element.raml",
+      "oas30/union-nil-plus-element.yaml",
+      Raml10YamlHint,
+      Oas30YamlHint
+    )
   }
 
   test("RAML with union of nil + multiple types to OAS 3.0") {
-    compatibility("raml10/union-nil-plus-elements.raml", "oas30/union-nil-plus-elements.yaml", Raml10YamlHint, Oas30YamlHint)
+    compatibility(
```

Comment:
Avoid adding formatting changes in the commit please

Reply:
sorry I don't like unformatted code, but didn't expected it would make the code hard to read for you

Reply:
please add it in another commit.

Reply:
done